packages/block-library/src/image/deprecated.js

```diff
@@ -941,4 +941,213 @@ const v7 = {
 	},
 };
 
-export default [ v7, v6, v5, v4, v3, v2, v1 ];
+const v8 = {
```

Comment:
The migration looks good. We have tests for each of them that are pretty easy to write by following the steps in [Full post content test fixtures](https://github.com/WordPress/gutenberg/blob/HEAD/test/integration/fixtures/blocks/README.md#creating-fixtures).

Reply:
Thanks! 🙂 

I've just added the fixtures in c18e283e3092d17520fc376bf9169297acabb814